husky_bringup/scripts/install

```diff
@@ -23,4 +23,22 @@ if os.environ.get('HUSKY_UR5_IP'):
 if os.environ.get('HUSKY_LMS1XX_IP'):
   j.add(package="husky_bringup", glob="launch/lms1xx_config/*")
 
+if os.environ.get('HUSKY_DUAL_UR5_ENABLED'):
+  j.add(package="husky_bringup", glob="launch/dual_ur5_arm_config/*")
+
+if os.environ.get('FLIR_PTU_ENABLED'):
+  j.add(package="husky_bringup", glob="launch/flir_ptu/*")
+
+if os.environ.get('BUMBLEBEE_ENABLED'):
+  j.add(package="husky_bringup", glob="launch/bumblebee/*")
+
+if os.environ.get('ROBOTIQ_GRIPPERS_ENABLED'):
+  j.add(package="husky_bringup", glob="launch/robotiq_s_model_grippers/*")
+
+if os.environ.get('VELODYNE_VLP16_ENABLED'):
+  j.add(package="husky_bringup", glob="launch/velodyne/*")
+
+if os.environ.get("ROBOTIQ_FT_300_ENABLED"):
```

Comment:
This adds a myriad of sensors available to be installed via the install script (for now ,it works only for the dual ur5 arm config, but in the future can be abstracted to set positions, xyz, rpy, for each respective sensor.. moving us to a more general robot configuration that is happily met by only touching env vars